internal/config/config.go

```diff
@@ -16,16 +18,38 @@ const (
 )
 
 type Config struct {
-	Server Server  `yaml:"server" json:"server"` // configuration of the public REST server
-	Name   string  `yaml:"name" json:"name"`     // used for OTEL as an application identifier
-	RqLite *RqLite `yaml:"rqlite" json:"rqlite"`
+	Server  Server  `yaml:"server" json:"server"` // configuration of the public REST server
+	Name    string  `yaml:"name" json:"name"`     // used for OTEL as an application identifier
+	Cluster Cluster `yaml:"cluster" json:"cluster"`
+}
+
+// TODO: clean up cluster & rqlite configuration
+type Cluster struct {
+	// BootstrapExpect sets expected number of servers to join into the cluster before bootstrap is called
+	BootstrapExpect int `yaml:"bootstrapExpect" json:"bootstrapExpect" env:"CLUSTER_BOOTSTRAP_EXPECT"`
+	// Bootstrap
+	Bootstrap bool    `yaml:"bootstrap" json:"bootstrap" env:"CLUSTER_BOOTSTRAP"`
+	RaftAddr  string  `yaml:"raftAddress" json:"raftAddress" env:"CLUSTER_RAFT_ADDR" env-default:":8090"`
```

Comment:
may I propose to a) keep the names consistent: ADDRESS and not mixed ADDR&ADDRESS?
I assume this is the bind address ... I prefer explicit naming
```suggestion
	RaftBindAddress string  `yaml:"raftBindAddress" json:"raftBindAddress" env:"CLUSTER_RAFT_BIND_ADDRESS" env-default:":8090"`
```

Reply:
this is only temporary (will get renamed into something like InternalAddr or something similar in the future)